src/NewCommand.php

```diff
@@ -91,6 +147,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln('<comment>Application ready! Build something amazing.</comment>');
     }
 
+
+    public function doesPresetFileExist()
```

Comment:
Is this method supposed to be completed?

Reply:
This is just some pre-abstraction before i made the decision to seek feedback first.